src/pages/ForgotPassword/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Input from 'components/form/Input';
+import Button from 'components/common/Button';
+import { useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import useTranslation from 'hooks/useTranslation';
+
+import './style.css';
+import { useResetPasswordMutation } from 'services/profile/profile';
+
+const ForgotPassword = () => {
+  const t = useTranslation();
+  const schema = z.object({
+    email: z.string().email({ message: t('signup.errors.emailMsg') }),
+  });
+  const {
+    register: forgotPassowrd,
+    handleSubmit: handleOnSubmitForgotPassowrd,
+    formState: { errors: forgotPassowrdErrors },
+  } = useForm({ resolver: zodResolver(schema) });
+  const [isSend, setIsSend] = useState(false);
+  const [resetPassword] = useResetPasswordMutation();
+  const redirectUrl = 'https://localhost:3000/login';
+  const onSubmitForgotPassowrd = data => {
+    if (!isSend) {
+      resetPassword({ ...data, redirect_url: redirectUrl })
+        .then(data => {
```

Comment:
And instead of using the `then()`,  `catch()` you could set the state in an useEffect:

```
  useEffect(() => {
    if (isSuccess) {
      setIsSend(true)
    }
  }, [isSuccess])
```